src/platform/plugins/shared/console/public/application/containers/embeddable/console_resize_button.tsx

```diff
@@ -29,29 +29,13 @@ export interface EmbeddedConsoleResizeButtonProps {
   setConsoleHeight: React.Dispatch<React.SetStateAction<number>>;
 }
 
-const parseOrDefaultValue = (value: string, defaultValue: number): number => {
-  try {
-    const result = parseInt(value, 10);
-    if (!isNaN(result) && result >= 0) {
-      return result;
-    }
-  } catch {
-    // ignore bad values
-  }
-  return defaultValue;
-};
-
 export function getCurrentConsoleMaxSize(euiTheme: EuiThemeComputed<{}>) {
-  const euiBaseSize = parseOrDefaultValue(euiTheme.size.base, 16);
-  const winHeight = window.innerHeight;
-  const bodyStyle = window.getComputedStyle(document.body);
-  const headerOffset = parseOrDefaultValue(
-    bodyStyle.getPropertyValue('--euiFixedHeadersOffset') ?? '0px',
-    0
-  );
+  const euiBaseSize = euiTheme.base;
+  const appRect = document.getElementById('app-fixed-viewport')?.getBoundingClientRect();
```

Comment:
This is a bit hacky, but it's more reliable than parsing the CSS variable. The problem with parsing is that it won't work if the CSS variable has a `calc(...)` expression inside.

Reply:
would adopting the following approach without having to parse said variable using purely CSS still work `max(var(--said-variable) - euiBaseSize, CONSOLE_MIN_HEIGHT)`?

Reply:
Hmm, I'm not sure. I think we need the current height in JavaScript for the existing code to work. From what I understand, the most reliable way to get the current value from a CSS variable in JavaScript is by using the `el.getBoundingClientRect();` approach.


Maybe the whole code can be refactored to avoid needing the CSS value in JavaScript. (I'm not sure about this). but it would require more work.